configuration-modes/appsensor-configuration-stax/src/test/resources/appsensor-server-config.xml

```diff
@@ -29,152 +29,191 @@
 			<client-application-name>server5</client-application-name>
 		</correlated-client-set>
 	</correlation-config>
+	
+	
 
-	<detection-points>
```

Comment:
It is for testing whether we can get a custom detection point into the config and verify against the xsd structure. It is basically for a test scenario.